build/npm.js

```diff
@@ -42,10 +42,15 @@ const npx = async (...args) => {
     await executeProcess("npx", ...args);
 };
 
-async function npmInstall(name) {
+async function npmInstall(name, offLine = true) {
+    let args = ["--no-audit"];
+    if (offLine) {
```

Comment:
Nit: You have `offlineBuild` above, not `offLineBuild`, so it should probably be `offline` here? I personally don't say "I'm off line", so seems about right for it to be a single word.